test/local/account_routes.js

```diff
@@ -6,7 +6,7 @@ require('ass')
 
 var sinon = require('sinon')
 
-var test = require('../ptaptest')
+var test = require('tap').test
```

Comment:
Noticed that not all the test files have migrated to `require('tap').test`, should we do a follow on issue?